index.html

```diff
@@ -30,37 +31,61 @@ <h1 class="detail">I’m a software developer</h1>
         <h1 class="recent">My Recent<br> Works</h1>
         <img src="./src/Indicator.png" alt="seperator">
     </section>
+
     <section id="my-projects" class="layout">
     <div class="articles">
-        <div class="project-logo">1</div>
-        <div class="project-name">2</div>
-        <div class="project-lang">3</div>
-        <div class="project-lang">4</div>
+        <section class="flexible-items-inside-grid"><img class="project-logo" src="./cards/prjct_logo.png" alt="logo"></section>
+        <section class="flexible-items-inside-grid"><h2 class="project-name">Multi-Post Stories<br>Gain+Glory</h2></section>
+        <section class="flexible-items-inside-grid">
+            <ul class="languages">
+                <li><button class="lang-btns">Ruby on rails</button></li>
+                <li><button class="lang-btns">css</button></li>
+                <li><button class="lang-btns">JavaScript</button></li>
+                <li><button class="lang-btns">html</button></li>
+                </ul>
+        </section>
+        <section class="flexible-items-inside-grid"><button class="prjct-btn">See Project</button></section>
     </div>
+
     <div class="articles">
-        <a>1</a>
-        <a>2</a>
-        <a>3</a>
+        <section class="flexible-items-inside-grid"><img class="project-logo" src="./cards/prjct_logo.png" alt="logo"></section>
+        <section class="flexible-items-inside-grid"><h2 class="project-name">Multi-Post Stories<br>Gain+Glory</h2></section>
+        <section class="flexible-items-inside-grid">
+            <ul class="languages">
+                <li><button class="lang-btns">Ruby on rails</button></li>
+                <li><button class="lang-btns">css</button></li>
+                <li><button class="lang-btns">JavaScript</button></li>
+                <li><button class="lang-btns">html</button></li>
+                </ul>
+        </section>
+        <section class="flexible-items-inside-grid"><button class="prjct-btn">See Project</button></section>
     </div>
+
     <div class="articles">
-        <a>1</a>
-        <a>2</a>
-        <a>3</a>
+        <section class="flexible-items-inside-grid"><img class="project-logo" src="./cards/prjct_logo.png" alt="logo"></section>
+        <section class="flexible-items-inside-grid"><h2 class="project-name">Multi-Post Stories<br>Gain+Glory</h2></section>
+        <section class="flexible-items-inside-grid">
+            <ul class="languages">
+                <li><button class="lang-btns">Ruby on rails</button></li>
+                <li><button class="lang-btns">css</button></li>
+                <li><button class="lang-btns">JavaScript</button></li>
+                <li><button class="lang-btns">html</button></li>
+                </ul>
+        </section>
+        <section class="flexible-items-inside-grid"><button class="prjct-btn">See Project</button></section>
     </div>
-
     </section>
```

Comment:
- [ ] This section is styled using `display: grid`. Please kindly fix that.

Reply:
I don't understand this error the section at line 55 
is a <div class="articles"> and it is styled using grid 
![image](https://user-images.githubusercontent.com/83329154/129424139-2c2fbea4-636f-4f54-9f9f-05315dec4864.png)
but the sections inside it each takes a grid portion and styled using flexbox
so what should I do ?